components/supervisor/frontend/src/index.ts

```diff
@@ -56,6 +57,14 @@ const toStop = new DisposableCollection();
         });
     }
     const supervisorServiceClient = new SupervisorServiceClient(gitpodServiceClient);
+    const ideMetricsServiceClient = new IDEMetricsServiceClient()
+
+    window.onerror = (event: Event | string, source?: string, lineno?: number, colno?: number, error?: Error) => {
+        // ignore error when report to ide metrics
+        ideMetricsServiceClient.addCounter("gitpod_workspace_browser_error_total").catch(() => {})
+        console.error(error)
```

Comment:
otherwise browser does not log it? just checking that we don't product duplicates